docs/src/integrate/patina_dxe_core_requirements.md

```diff
@@ -83,37 +85,19 @@ multiple of 4KB as a page size in order to apply image memory protections. The E
 memory protections on images without this section alignment requirement, but it will dispatch them, depending on
 configuration.
 
-ARM64 DXE_RUNTIME_DRIVERs must have a multiple of 64 KB image section alignment per UEFI spec requirements.
+AArch64 DXE_RUNTIME_DRIVERs must have a multiple of 64 KB image section alignment per UEFI spec requirements.
 This is required to boot operating systems with 16 KB or 64 KB page sizes.
 
 Patina components will have 4 KB section alignment by nature of being compiled into Patina.
 
 The DXE Readiness Tool validates all drivers have a multiple of 4 KB section alignment and reports an error if
-not. It will also validate that ARM64 DXE_RUNTIME_DRIVERs have a multiple of 64KB section alignment.
+not. It will also validate that AArch64 DXE_RUNTIME_DRIVERs have a multiple of 64KB section alignment.
 
 > **Guidance:**
 > All C based drivers must be compiled with a linker flag that enforces a multiple of 4 KB section alignment.
-> For MSVC, this linker flag is `/ALIGN:0x1000` for GCC/CLANG, the flag is `-z common-page-size=0x1000`. This section
-> allows for multiples of 4 KB or 64 KB, depending on driver, but unless a specific use case dictates greater section
-> alignment, then it is recommended to use 4 KB for everything except for ARM64 DXE_RUNTIME_DRIVERs, which should use
-> 64 KB, e.g. `/ALIGN:0x10000` for MSVC and `-z common-page-size=0x10000` for GCC/CLANG.
-
-#### 1.4 CpuDxe Is No Longer Used
-
-EDK II supplies a driver named `CpuDxe` that provides CPU related functionality to a platform. In Patina DXE Core, this
-is part of the core, not offloaded to a driver. As a result, the CPU Arch and memory attributes protocols are owned by
-the Patina DXE Core. MultiProcessor (MP) Services are not part of the core. ARM64 already does not have MP Services
-owned by `CpuDxe`, `ArmPsciMpServicesDxe` owns them. There is an x64 implementation of
-[`MpDxe`](https://github.com/microsoft/mu_basecore/blob/HEAD/UefiCpuPkg/MpDxe/MpDxe.inf) that platforms should add to
-their flash file for use with the Patina DXE Core if MP services are desired.
-
-On ARM64 systems, when Patina assumes ownership of `CpuDxe` it also encompasses the functionality provided by
-`ArmGicDxe`. This is because the GIC is considered a prerequisite for `CpuDxe` to handle interrupts correctly. As such,
-ARM64 platforms also should not include `ArmGicDxe`.
-
-> **Guidance:**
-> Platforms must not include `CpuDxe` in their platforms and instead use CPU services from Patina DXE Core and MP Services
-> from a separate C based driver. ARM64 platforms should not use `ArmGicDxe`.
+> For MSVC, this linker flag is `/ALIGN:0x1000` for GCC/CLANG, the flag is `-z common-page-size=0x1000`. It is
```

Comment:
This is not entirely accurate, clang with lld-linker uses MSVC style linker args, e.g. /ALIGN:0x1000. Maybe easiest to say something like (and the same below):


```suggestion
> For MSVC/CLANGPDB, this linker flag is `/ALIGN:0x1000` for GCC/CLANGDWARF, the flag is `-z common-page-size=0x1000`. It is
```